Add Vue.js Roadtrip Berlin event data

gustojs opened the pull request:
Source: Twitter and https://www.eventbrite.co.uk/e/vuejs-roadtrip-berlin-tickets-43925232553

Commit:
Add Vue.js Roadtrip Berlin event data

Source: Twitter and https://www.eventbrite.co.uk/e/vuejs-roadtrip-berlin-tickets-43925232553

shentao reviewed (commented):
We should probably wait with the page getting updated with a mention about the Berlin event.

Commit:
Change website to Eventbrite

I've had a talk with the organizer and we decided to switch it to Eventbrite for now. There will be official website on http://www.vuejsroadtrip.com/berlin in a later time.

gustojs commented:
I've had a talk with the organizer and we decided to switch it to Eventbrite for now. There will be official website on http://www.vuejsroadtrip.com/berlin in a later time.
